06-building-a-program-start-to-finish/writing-tests/output/InvertedIndexSpec.js

```diff
@@ -0,0 +1,400 @@
+//  book with valid contents
+const book = require('./allBooks/books.json');
+const secondBook = require('./allBooks/newBook.json');
+const myBook = require('./allBooks/myBook.json');
+//  book with invalid content
+const invalidFile = require('./allBooks/invalid.json');
+//  empty book
+const empty = require('./allBooks/empty.json');
+// an array
+const invalid = require('./allBooks/in-valid.json');
+// empty array
+const emptyFile = require('./allBooks/emptyArray.json');
+// an invalid JSON file
+const notValid = require('./allBooks/notArrayOfArray.json');
+
+const meek = new InvertedIndex();
+
```

Comment:
Is this variable name (**_meek_**) appropriate for this instance of your InvertedIndex? Please change it to whats appropriate then I will complete the review of your test.